test-complete-app/build.gradle

```diff
@@ -0,0 +1,16 @@
+plugins {
+	id "net.saliman.properties" version "1.5.1"
+  id "com.marklogic.ml-gradle" version "4.7.0"
+}
+
+/*
+ * Since our REST API server uses a certificate template, we need to execute an extra command to generate a temporary
+ * certificate for this template - without that, the REST API server won't be able to receive HTTP or HTTPS requests.
+ */
+ext {
+    def command = new com.marklogic.appdeployer.command.security.GenerateTemporaryCertificateCommand()
+    command.setTemplateIdOrName("ssl-project-template")
```

Comment:
Use a different name for the template, like `mlxprs-ssl-template`, so it's obvious if someone is deploying this and other project's test apps to the same cluster what this is intended for.